config/urls.py

```diff
@@ -191,6 +191,14 @@
         "robots.txt",
         TemplateView.as_view(template_name="robots.txt", content_type="text/plain"),
     ),
+    path(
+        ".well-known/trust.txt",
+        TemplateView.as_view(template_name="trust.txt", content_type="text/plain"),
+    ),
+    path(
+        "trust.txt",
+        RedirectView.as_view(url="/.well-known/trust.txt"),
```

Comment:
Looking at the [spec](https://journallist.net/reference-document-for-trust-txt-specifications) I think this is actually backwards - we should redirect from `/.well-known/trust.txt` to `/trust.txt`.